runtime/parser/expression.go

```diff
@@ -1262,7 +1262,7 @@ func defineIndexExpression() {
 				firstIndexExpr,
 				ast.NewRange(
 					p.memoryGauge,
-					token.StartPos,
+					left.StartPosition(),
```

Comment:
now that `token` parameter is not used anymore, we can omit declaring this unused parameter by using the underscore (`_`) as the parameter name.
i.e: in the function signature above:
```go
func(p *parser, _ lexer.Token, left ast.Expression) (ast.Expression, error) {
```

Reply:
`token` is now omitted.